staging/src/k8s.io/kubectl/pkg/cmd/util/editor/launcher.go

```diff
@@ -0,0 +1,70 @@
+//go:build !windows
```

Comment:
nit: can we rename this file to `launcher_others.go`?

Reply:
Sure

Reply:
Technically, this is the default launcher and Windows is the exception, though, right?

How do we name files other places where there is an OS-specific version of something?  I don't recall offhand and I'm on mobile right now or I would look myself (and can later), but I'm just posing the question.

Reply:
Yes, this is just exception to Windows and `cmd` in particular.

I see `*_others.go` in the tree elsewhere, like `pkg/kubelet/eviction/defaults_{linux,others}.go`, `pkg/kubelet/nodeshutdown/systemd/inhibit_{linux,others}.go`, `/cmd/kubelet/app/server_{linux,others,windows}.go`. So this looks reasonable.